SU2_CFD/src/driver_structure.cpp

```diff
@@ -2781,27 +2854,48 @@ unsigned short CDriver::GetMovingMarker() {
 
 unsigned long CDriver::GetNumberVertices(unsigned short iMarker) {
 
-  unsigned long nFluidVertex;
+  unsigned long nVertices(0);
```

Comment:
Although it is not wrong, but maybe use an assignment here, so that it is consistent with the rest of the code.